### DISCUSSION TITLE
Can I keep plotting/farming in Chia while mining whit Hpool

### DISCUSSION THREAD

**Discussion by Entretoize** [Chia-Network/chia-blockchain#6608] [Support]
I have enough space to have hundred of plot but it takes much time, also my first plot is ready to farm, then can I start HPool miner while I keep Chia blockchain running as it is also farming ? Or will I get banned ?
Thanks

**Comment by rmica** [comment]
Can (must) plot. Can't farm.

**Comment by loppefaaret** [comment]
you have to ask hpool about wether that is something they dont like or not, they are running closed source code, that is nothing we can do anything about!

**Comment by Entretoize** [comment]
I finally blocked start_farmer.exe in my firewall, this way ready plots are not detected as valid and not farmed in chia app and I can let hpool miner run on these plots while continuing to plot new plots.

**Reply by DavidJ68** [reply]
cheater

**Reply by Entretoize** [reply]
?? It's not cheating, I plot with chia and farm with hpool.

**Comment by BrandtH22** [comment]
@Entretoize , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
